pyproject.toml

```diff
@@ -1273,7 +1273,10 @@ extend-select = [
     "D2",
     "D3",
     "D400",
-    # "D401", # Not enabled by ruff, but we don't want it
```

Comment:
I'm just confused by this comment. I guess we finally want to enable this rule, don't we?

Reply:
I think it was some kind of copy & paste. 

BTW. I do not think it's a high prority to fix all those, but it could be a great good-first-issue for people, so maybe we should create such a good first issue specifically for D401 after we merge this one?

Reply:
Yeah, good idea!

Reply:
It's already in [this issue](https://github.com/apache/airflow/issues/10742), do we need another?

Reply:
After this PR merged we would have exact list of the modules which should be fixed.
So final completion might be tracked into the #10742, and per module (i guess per provider) might be tracked into the separate one, like a sub-task.